README.md

```diff
@@ -37,52 +37,79 @@ from App Runner services.
+# HELP ecs_container_network_transmit_errors_total Cumulative total count of container network errors in transmit.
+# TYPE ecs_container_network_transmit_errors_total counter
+ecs_container_network_transmit_errors_total{container_name="ecs-exporter",interface="eth1"} 0
+# HELP ecs_container_network_transmit_packets_total Cumulative total count of container network packets transmitted.
+# TYPE ecs_container_network_transmit_packets_total counter
+ecs_container_network_transmit_packets_total{container_name="ecs-exporter",interface="eth1"} 713
+# HELP ecs_exporter_build_info A metric with a constant '1' value labeled by version, revision, branch, goversion from which ecs_exporter was built, and the goos and goarch for the build.
+# TYPE ecs_exporter_build_info gauge
+ecs_exporter_build_info{branch="",goarch="arm64",goos="linux",goversion="go1.23.2",revision="unknown",tags="unknown",version=""} 1
+# HELP ecs_task_cpu_limit_vcpus Configured task CPU limit in vCPUs (1 vCPU = 1024 CPU units). This is optional when running on EC2; if no limit is set, this metric has no value.
+# TYPE ecs_task_cpu_limit_vcpus gauge
+ecs_task_cpu_limit_vcpus 0.25
+# HELP ecs_task_ephemeral_storage_allocated_bytes Configured Fargate task ephemeral storage allocated size in bytes.
+# TYPE ecs_task_ephemeral_storage_allocated_bytes gauge
+ecs_task_ephemeral_storage_allocated_bytes 2.1491613696e+10
+# HELP ecs_task_ephemeral_storage_used_bytes Current Fargate task ephemeral storage usage in bytes.
+# TYPE ecs_task_ephemeral_storage_used_bytes gauge
+ecs_task_ephemeral_storage_used_bytes 3.7748736e+07
+# HELP ecs_task_image_pull_start_timestamp_seconds The time at which the task started pulling docker images for its containers.
+# TYPE ecs_task_image_pull_start_timestamp_seconds gauge
+ecs_task_image_pull_start_timestamp_seconds 1.7291179014941156e+09
+# HELP ecs_task_image_pull_stop_timestamp_seconds The time at which the task stopped (i.e. completed) pulling docker images for its containers.
+# TYPE ecs_task_image_pull_stop_timestamp_seconds gauge
+ecs_task_image_pull_stop_timestamp_seconds 1.7291179144469e+09
+# HELP ecs_task_memory_limit_bytes Configured task memory limit in bytes. This is optional when running on EC2; if no limit is set, this metric has no value.
+# TYPE ecs_task_memory_limit_bytes gauge
+ecs_task_memory_limit_bytes 5.36870912e+08
+# HELP ecs_task_metadata_info ECS task metadata, sourced from the task metadata endpoint version 4.
+# TYPE ecs_task_metadata_info gauge
+ecs_task_metadata_info{availability_zone="us-east-1a",cluster="arn:aws:ecs:us-east-1:829490980523:cluster/prom-ecs-exporter-sandbox",desired_status="RUNNING",family="prom-ecs-exporter-sandbox-isker-fargate",known_status="RUNNING",launch_type="FARGATE",pull_started_at="2024-10-16T22:31:41.494115693Z",pull_stopped_at="2024-10-16T22:31:54.446899683Z",revision="11",task_arn="arn:aws:ecs:us-east-1:829490980523:task/prom-ecs-exporter-sandbox/0c7f6b0414dc47d0a15019a099cd919b"} 1
```

Comment:
Now that we have metrics for them, let's remove `pull_started_at` and `pull_stopped_at` from the metadata labels.